webrtc.html

```diff
@@ -2783,6 +2783,29 @@
           <dd>
             <p>An array containing parameters for RTP encodings of media.</p>
           </dd>
+
+          <dt>sequence&lt;RTCRtpCodecParameters&gt; codecs</dt>
+
+          <dd>
+            <p>An array containing codecs that may be used to send RTP
+               encodings.  These can be read
+               from <code>RTCRtpSender.getParameters</code>, and can
+               be changed by <code>RTCRtpSender.setParameters</code>,
+               but the only changes allowed are to either reorder the
+               codecs or remove codecs.  Codecs may not be added or
+               changed.
+            </p>
+            <p>If codecs are reordered, the new order indicates the
+            preference for sending, with the first codec being the
+            codec with highest preference.  If a codec is removed,
+            that codec will not be used to send.  The effect of
+            reordering or removing codecs last until the codec are
+            renegotiated via <code>setLocalDescription</code>
+            and <code>setRemoteDescription</code>.  After the codecs
```

Comment:
I think it's enough to talk about negotiation here. I.e., skip the references to setLocal/RemoteDescription(). If you really need to keep them, they should be linked.